src/akvo_reflow/endpoint/reload.clj

```diff
@@ -0,0 +1,12 @@
+(ns akvo-reflow.endpoint.reload
+  (:require [akvo-reflow.main :as main]
+            [clojure.java.shell :as shell]
+            [compojure.core :refer [routes POST]]))
+
+(defn reload-endpoint [config]
+  (routes
+   (POST "/" []
+     (let [pull (shell/with-sh-dir (:flow-server-config main/config) ;; FIXME
```

Comment:
I would like @zzgvh to fix that part :wink: e.g. getting the config from the param passed to the endpoint